### PR TITLE
Add deeper travis tests

### PR DESCRIPTION
**Pull request by natebosch**
Check analyzer, tests, and format

Format the files which are stale against current dart_style

### CODE REVIEW

File: tool/travis.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail fast
+set -e
+
+# Analyze
+dartanalyzer --fatal-warnings lib/*dart lib/messages/*dart test/*dart
+
+# Test
+pub run test:test
```
Comment (jakemac53) [inline tool/travis.sh]: nit, you shouldn't need the `:test` here

File: tool/travis.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail fast
+set -e
+
+# Analyze
+dartanalyzer --fatal-warnings lib/*dart lib/messages/*dart test/*dart
```
Comment (jakemac53) [inline tool/travis.sh]: any reason we aren't just doing everything under lib?

File: tool/travis.sh (hunk 1 above)
Comment (natebosch) [inline tool/travis.sh]: done

File: tool/travis.sh (hunk 2 above)
Comment (natebosch) [inline tool/travis.sh]: doing everything under lib is potentially redundant. The analyzer will hit all the code that is reachable from the files under lib/*dart which includes the stuff under src. For example running it today we see hints for src/resolver_impl.dart

If I change the glob to something like lib/**/*.dart we end up seeing duplicate hints

Comment (jakemac53) [inline tool/travis.sh]: ohh right I remember that now

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**